docs/bnb_smart_chain/staking/index.md

```diff
@@ -0,0 +1,191 @@
+
+### Validator Election
+
+There are different roles for validators:
+
+* Cabinet: the top K (which is 21 currently) validators who get the most chance of producing blocks.
+* Candidate: the top (K, K+NumOfCandidates] validators who get a small chance of producing blocks.
+* Inactive: the reset validators who get no chance of producing blocks.
+
+![img](../img/staking/validator-election.png)
+
+The validator set roles are determined every 24 hours based on the latest staking information.
+
+After UTC 00:00, the consensus engine sorts validators and updates the BSC validator set with the ranking information.
+
+### System Contracts
+
+There are several built-in contracts (i.e., system contracts) to facilitate the BSC staking.
+
+* Validator Set Contract. The contract periodically elects a validator set.
+  The contract also serves as a vault for temporarily storing validator rewards.
+
+* System Reward Contract. This contract acts as a vault to collect part of transaction fees. The funds are used for
+  various public purposes, like distributing fast finality rewards.
+
+* Slash Contract. This contract is used to keep track of the number of times a validator becomes unavailable and
+  triggers penalties once a certain threshold is reached. Additionally, this contract also handles other types of slash
+  events, such as double signing and malicious voting in fast finality.
+
+* Stake Hub Contract. This contract serves as the entry point for managing validators and delegations,
+  while also implementing the logic for slashing specific validators. For delegation/undelegation/redelegation
+  operations, it will call different validators' implementation contracts to manage a user's stake.
+
+### Credit Contract
+
+Each validator has its own validator contract that manages staking credit and facilitates the exchange
+between credit and BNB. The token name of a staking credit is "stake {{validator moniker}} credit",
+and the symbol is "st{{validator moniker}}". The contract will be created by the Stake Hub Contract when a validator
+is created.
+
+### Reward Distribution
+
+The staking reward comes from transaction fee - when a block is produced, the majority of the block fee will be
+collected as reward for the validator who proposed the block.
+
+Every day, a portion of the rewards collected will be directly sent to the operator account of the validator as
+commission, while the remaining portion will be sent to the corresponding validator credit contract. And when a user
+undelegates and claims his/her stakes, the accumulated reward and the original stake will be sent back to him/her.
+
+## Validator Operations
+
+Validators are nodes running BNB Smart Chain software, participating in the consensus process.
+They require a minimum BNB stake at their validator address and can receive delegations from other BNB holders.
+Validators earn rewards from transaction fees and share a part of these rewards with their delegators.
```

Comment:
==> `Validators earn rewards from transaction fees and share most of these rewards with their delegators.`

Reply:
fixed